cmd/entrypoint/main.go

```diff
@@ -173,8 +160,13 @@ func main() {
 	}
 
 	if err := e.Go(); err != nil {
-		breakpointExitPostFile := e.PostFile + breakpointExitSuffix
 		switch t := err.(type) {
+		case entrypoint.DebugBeforeStepError:
+			log.Println("Skipping step because before step breakpoint fail-continue")
```

Comment:
The `entrypoint.DebugBeforeStepError` error is caused by the user executing the script `debug-beforestep-fail.sh`, indicating that the user has chosen to abort execution of that step program.

i changed the log message to `Skipping execute step script because before step breakpoint fail-continue`